### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_Top Operations_Keyboard: We are not able to access column headers of the "Top Operations" grid by a keyboard.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/azuredatastudio#20779]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #Keyboard; #WCAG2.1.1; #FTP;

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:
1. Launch the "Azure data studio" application.
2. Connect with the provided SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. Tab to "Top operations" control from the right navigation.
7. Observe that We are not able to access column headers of the "Top Operations" grid by a keyboard.

### Actual:
We are not able to access column headers of the "Top Operations" grid by a keyboard.

**NOTE: Similar issue is observed throughout the application.**

### Expected Result:
The column headers of the "Top Operations" grid should be accessible by a keyboard.

### Attachment:
![A11y_Azure Data Studio_Toolbar_Top Operations_Keyboard](https://user-images.githubusercontent.com/115216656/194815195-52206336-d2f0-4862-a070-298cedf42662.gif)

**Comment by aasimkhan30** [comment]
@Rama8345, we have added a keyboard shortcut F3 to sort the rows. This is an alternate solution of directly accessing the column headers. Can you please re test and close this issue.

**Comment by Rama8345** [comment]
Verified the bug in the latest insiders build, but we are getting below error.
![image](https://user-images.githubusercontent.com/115216656/199224487-d33530f1-bcbe-4564-9702-ef0e22d2b896.png)

**Comment by Rama8345** [comment]
Verified the issue in latest insider build. The issue is fixed now we are able to sort and resize column width with the keyboard. So closing the bug. Please find below attachment for reference.
![20779_Closed](https://user-images.githubusercontent.com/115216656/204995114-0864e239-2a3f-468a-b4b7-e02b49572d69.gif)

**Comment by 12-shweta** [comment]
**Github Tags:**#Closed;
